### PR TITLE
Updated Installer

### PR DESCRIPTION
**Pull request by velkymx**
The installer has been updated to remove all of the setup for email, docker and cron to satisfy #1483

### CODE REVIEW

File: ProcessMaker/Console/Commands/Install.php [hunk 1]
```diff
@@ -153,36 +141,23 @@ public function handle()
...
-        
-        // Install Docker executors
-        $this->info(__('Installing Docker executors...'));        
-        $this->installDockerImages();
-        
         // Restart services so they pick up the new settings
-        $this->info(__('Restarting Services...'));
-        $this->info(
+        $this->info(__('Restarting services...'));
+        $this->info(e
```
Comment (ryancooley) [inline ProcessMaker/Console/Commands/Install.php]: Extra character "e" is causing some issues here
```suggestion
        $this->info(
```

Comment (ryancooley) [reply ProcessMaker/Console/Commands/Install.php]: Other than that change above this looks good and functions as intended.

Comment (tdondich) [reply ProcessMaker/Console/Commands/Install.php]: No services should be handled by the installer.  What's it trying to restart?

Comment (velkymx) [reply ProcessMaker/Console/Commands/Install.php]: I've removed it

**Pull request merged**